stream-chat-android-client/src/main/java/io/getstream/chat/android/client/utils/ProgressCallback.kt

```diff
@@ -2,8 +2,37 @@ package io.getstream.chat.android.client.utils
 
 import io.getstream.chat.android.client.errors.ChatError
 
+/**
+ * Callback to listen for file upload status.
+ */
 public interface ProgressCallback {
-    public fun onSuccess(file: String)
+
+    /**
+     * Called when the attachment is uploaded successfully with an [attachmentUrl].
+     */
+    public fun onSuccess(attachmentUrl: String?)
+
+    /**
+     * Called when the attachment could not be uploaded due to cancellation, network problem or timeout etc with an [error].
+     *
+     * @see ChatError
+     */
     public fun onError(error: ChatError)
+
+    /**
+     * Called when the attachment upload is in progress with [progress] in percentage.
```

Comment:
Is this comment true? We do pass the `progress` as a `Long` value, not sure how that's the percentage.

Reply:
Yeah, we used to do the same earlier. Of course, it doesn't make sense.